src/Microsoft.ML.Transforms/ProjectionCatalog.cs

```diff
@@ -36,7 +36,7 @@ public static class ProjectionCatalog
         /// </summary>
         /// <param name="catalog">The transform's catalog.</param>
         /// <param name="columns">The input columns to use for the transformation.</param>
-        public static RandomFourierFeaturizingEstimator CreateRandomFourierFeatures(this TransformsCatalog.ProjectionTransforms catalog, params RandomFourierFeaturizingTransformer.ColumnInfo[] columns)
+        public static RandomFourierFeaturizingEstimator CreateRandomFourierFeatures(this TransformsCatalog.ProjectionTransforms catalog, params RandomFourierFeaturizingEstimator.ColumnInfo[] columns)
```

Comment:
Needs a Remarks section with details about the calculation.

Reply:
Same here.

---
In reply to: [254014739](https://github.com/dotnet/machinelearning/pull/2366#discussion_r254014739) [](ancestors = 254014739)